src/App.tsx

```diff
@@ -15,36 +16,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
> Use `enum SortType` instead of string literals for possible sorting options (including the default one)

Based on the task requirements, it is better to fix them.